registrations/permissions.py

```diff
@@ -4,24 +4,25 @@
 
 from events.auth import ApiKeyUser
 from events.permissions import UserDataFromRequestMixin
-from registrations.models import Registration, SignUp
+from registrations.models import Registration, SignUp, SignUpGroup
 
 
 class CanAccessRegistration(UserDataFromRequestMixin, permissions.BasePermission):
     def has_permission(self, request: Request, view: APIView) -> bool:
         if not request.user.is_authenticated:
             return False
 
+        if request.user.is_superuser:
+            return True
+
         if request.method == "POST":
             (
                 __,
                 user_organization,
             ) = self.user_data_source_and_organization_from_request(request)
-            return (
-                request.user.is_superuser
-                or request.user.is_admin_of(user_organization)
-                or request.user.is_registration_admin_of(user_organization)
-            )
+            return request.user.is_admin_of(
+                user_organization
+            ) or request.user.is_registration_admin_of(user_organization)
```

Comment:
The requirements were updated after this PR, but all authenticated users should be able to 
- create signups and signup groups
- view, update and delete signups created by themselves

See PR: https://github.com/City-of-Helsinki/linkedevents/pull/719
And updated documentation: https://helsinkisolutionoffice.atlassian.net/wiki/spaces/KAN/pages/8007483434/K+ytt+oikeustasot+ilmoittautuminen

Reply:
We can perhaps merge your PR first to main and I'll resolve this accordingly after that

Reply:
Now changed after rebasing to main